src/main/webapp/script.js

```diff
@@ -33,9 +35,47 @@ function init() {
 /**
  * Compiles the handlebars template representing a task.
  */
-function compileTaskTemplate() {
+function compileTemplates() {
   const taskTemplateElement = document.getElementById("task-template").innerHTML;
+  const selectOptionsTemplateElement = document.getElementById("select-options-template").innerHTML;
   taskTemplate = Handlebars.compile(taskTemplateElement);
+  selectOptionsTemplate = Handlebars.compile(selectOptionsTemplateElement);
+}
+
+const DEFAULT_WORKING_HOURS = {
+  "start-hour": "09",
+  "start-min": "00",
+  "end-hour": "18",
+  "end-min": "00"
+}
+
+function addZeroBefore(n) {
+  return (n < 10 ? '0' : '') + n;
+}
+
+function initSettings() {
+  workingHours = DEFAULT_WORKING_HOURS;
+
+  if(localStorage.hasOwnProperty("workingHours")){
+    workingHours = JSON.parse(localStorage.getItem("workingHours"));
+  }
+  for(settingId in workingHours){
+    $(`#${settingId} li[data-value=${workingHours[settingId]}]`).addClass("mdc-list-item--selected");
```

Comment:
I just noticed now that you're using inconsistent quoting all over the JS code. I recommend cleaning this up in a separate PR after to always use single quotes `'` or ticks `` ` `` for template strings.

Reply:
Yes, I have just learned about template strings, quickly checked if I saw other places they could be used, I haven't really found a place, but it is possible I missed it.